region/brinstar/red/Alpha Power Bomb Room.json

```diff
@@ -208,6 +208,122 @@
         ]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphPowerBomb",
+        {"or": [
+          "h_canArtificialMorphMovement",
+          {"and": [
+            "h_canArtificialMorphBombs",
+            "canBePatient"
+          ]}
+        ]}
+      ],
+      "note": [
+        "Traverse the room while avoiding or killing the Boyons.",
+        "If the Chozo item is still there, it is best to exit G-Mode before the final Power Bomb goes off to ensure that PLMs aren't overloaded.",
+        "If Samus falls into a Samus Eater, she will be teleported into it when exiting G-Mode."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks (Power Bombs Only)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphPowerBomb",
+        {"or": [
+          {"ammo": { "type": "PowerBomb", "count": 6}},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 4}},
+            "h_canArtificialMorphBombHorizontally"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 4}},
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 3}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }},
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 2
+            }}
+          ]}
+        ]}
+      ],
```

Comment:
```suggestion
      ],
      "clearsObstacles": ["A", "B"],
```

Reply:
I find 3 PBs usually kills 3 of the Boyons, and sometimes kill all 4.  4 Pbs should always kill them all.